model/sharing/member.go

```diff
@@ -301,10 +306,35 @@ func (s *Sharing) DelegateAddContacts(inst *instance.Instance, contactIDs map[st
 			Name:     name,
 			Email:    email,
 			Instance: cozyURL,
-			ReadOnly: ro,
+			ReadOnly: readOnly,
 		}
 		api.members = append(api.members, m)
 	}
+
+	for _, groupID := range groupIDs {
+		group, err := contact.FindGroup(inst, groupID)
+		if err != nil {
+			return err
+		}
+		g := Group{ID: groupID, Name: group.Name(), ReadOnly: readOnly}
+		api.groups = append(api.groups, g)
+
+		contacts, err := group.FindContacts(inst)
+		if err != nil {
+			return err
+		}
+		groupIndex := len(s.Groups)
+		for _, contact := range contacts {
+			m, err := buildMemberFromContact(contact, readOnly)
+			if err != nil {
+				return err
+			}
+			m.Groups = []int{groupIndex}
+			m.OnlyInGroups = true
```

Comment:
Why are we setting this to `true`?

Reply:
It is only used if there the contact is not already a member of the sharing.